functions.py

```diff
@@ -1,50 +1,43 @@
 from tkinter import Label
 import time, datetime, random
 import pyautogui
+from typing import Callable
 
 ########################################################
 
 def undock(x, y):
-
-    random_time = random.uniform(1,2)
     log("undocking...")
     # undock
-    pyautogui.moveTo(x,y, duration=random_time)
+    pyautogui.moveTo(x,y, duration=random.uniform(1,2))
     pyautogui.mouseDown(button='left')
-    sleep_and_log(random_time)
+    sleep_and_log(0.5)

```

Comment:
I need to make a higher order function that do the moveTo operation and then wait the same amount it used to move to the location. And then use that wherever i use moveTo and sleep